integration/v3_grpc_test.go

```diff
@@ -193,3 +194,130 @@ func TestV3DeleteRange(t *testing.T) {
 		clus.Terminate(t)
 	}
 }
+
+// TestV3WatchFromCurrentRevision tests Watch APIs from current revision.
+func TestV3WatchFromCurrentRevision(t *testing.T) {
+	tests := []struct {
+		putKeys        []string
+		watchRequest   *pb.WatchRequest
+		headerRevision int64
```

Comment:
can we remove this headerRevision? this is already included in wresp, no? Also please put a blank line above wresp to separate the input and expect values.